aws/table_aws_wellarchitected_check_detail.go

```diff
@@ -0,0 +1,315 @@
+			continue
+		}
+
+		input := &wellarchitected.ListCheckDetailsInput{
+			MaxResults: maxLimit,
+			LensArn:    aws.String(*answer.LensArn),
+			PillarId:   aws.String(*answer.PillarId),
+			QuestionId: aws.String(*answer.QuestionId),
+			WorkloadId: aws.String(*answer.WorkloadId),
+			ChoiceId:   aws.String(*choice.ChoiceId),
+		}
+
+		paginator := wellarchitected.NewListCheckDetailsPaginator(svc, input, func(o *wellarchitected.ListCheckDetailsPaginatorOptions) {
+			o.Limit = maxLimit
+			o.StopOnDuplicateToken = true
+		})
+
+		for paginator.HasMorePages() {
+			output, err := paginator.NextPage(ctx)
+			if err != nil {
+				if strings.Contains(err.Error(), "ResourceNotFoundException") || strings.Contains(err.Error(), "ValidationException") {
+					return nil, nil
+				}
+				return nil, err
+			}
+
+			for _, item := range output.CheckDetails {
+				d.StreamListItem(ctx, CheckDetailInfo{item, answer.WorkloadId})
+
+				if d.RowsRemaining(ctx) == 0 {
+					return nil, nil
+				}
+			}
+		}
+	}
+	return nil, nil
+}
+
+func getAnswerDetailsForWorkload(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) ([]AnswerInfo, error) {
+	workload := h.Item.(types.WorkloadSummary)
+	workloadId := workload.WorkloadId
+
+	// Validate - User inputs must not be blank and return nil if doesn't match the hydrated workload id
+	if d.EqualsQualString("workload_id") != "" && d.EqualsQualString("workload_id") != *workloadId {
+		return nil, nil
+	}
+
+	commonData, err := getCommonColumns(ctx, d, h)
+	if err != nil {
+		return nil, err
+	}
+
+	commonColumnData := commonData.(*awsCommonColumnData)
+
+	var answerList []AnswerInfo
+
+	// Create session
+	svc, err := WellArchitectedClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_wellarchitected_answer.getAnswerDetailsForWorkload", "client_error", err)
+		return nil, err
+	}
+
+	// Unsupported region, return no data
+	if svc == nil {
+		return nil, nil
+	}
+
+	for _, lensAlias := range workload.Lenses {
+		lensArn := lensAlias
+		if !isAWSARN(lensArn) {
+			// Format for AWS_OFFICIAL- arn:aws:wellarchitected::aws:lens/<lensAlias>
+			lensArn = "arn:" + commonColumnData.Partition + ":wellarchitected::aws:lens/" + lensAlias
+		}
+		if d.EqualsQualString("lens_arn") != "" && d.EqualsQualString("lens_arn") != lensArn {
+			continue
+		}
+
+		input := &wellarchitected.ListAnswersInput{
+			MaxResults: int32(50),
+			LensAlias:  aws.String(lensArn),
+			WorkloadId: aws.String(*workloadId),
+		}
+
+		paginator := wellarchitected.NewListAnswersPaginator(svc, input, func(o *wellarchitected.ListAnswersPaginatorOptions) {
+			o.Limit = int32(50)
+			o.StopOnDuplicateToken = true
+		})
+
+		// List call
+		for paginator.HasMorePages() {
+			output, err := paginator.NextPage(ctx)
+			if err != nil {
+				if strings.Contains(err.Error(), "ResourceNotFoundException") || strings.Contains(err.Error(), "ValidationException") {
```

Comment:
@karanpopat we should update the code to not ignore `ValidationException` and we should follow https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_wellarchitected_answer.go#L234-L239 for ignoring the `ResourceNotFoundException`.